ton_client/src/debot/dengine.rs

```diff
@@ -729,6 +771,22 @@ impl DEngine {
         routines::call_routine(self.ton.clone(), name, args, signer).await
     }
 
+    async fn handle_output(&mut self, mut output: RunOutput) -> ClientResult<()> {
+        for msg in std::mem::take(&mut output.interface_calls) {
+            // TODO: check if there are builtin interfaces
+            // BuiltinInterfaces::try_execute(&msg)
+            self.browser.send(msg.0).await;
+        }
+
+        // TODO: 
+        // result.send_msgs();
+
+        for _msg in std::mem::take(&mut output.get_method_calls) {
```

Comment:
You may use variable directly, without `std::mem::take`

Reply:
Fixed